### DISCUSSION TITLE
Running sample code shows no output

### DISCUSSION THREAD

**Discussion by Wosler-Menna** [awslabs/amazon-kinesis-video-streams-webrtc-sdk-c#2008] [Q&A]
I set up kinesis on windows 10 with visual studio community 2022 and after it builds without errors I tried running the sample code through ` .\kvsWebrtcClientMaster.exe testchannel ` but it quits immediately without printing any outputs or errors. Did anyone face a similar problem?
When running with gdb through this command:
` gdb --args .\kvsWebrtcClientMaster.exe testchannel`
This is the output I get:
```
(gdb) run
Starting program: C:\webrtc\build\samples\kvsWebrtcClientMaster.exe testchannel
[New Thread 18532.0x20dc]
[New Thread 18532.0x5d3c]
[New Thread 18532.0x7cdc]
[Thread 18532.0x3a84 exited with code 3221225781]
[Thread 18532.0x5d3c exited with code 3221225781]
[Thread 18532.0x20dc exited with code 3221225781]
During startup program exited with code 0xc0000135.
(gdb)
```

**Comment by stefankiesz** [comment]
Hi Menna, I haven't before seen such an error. Based on what I found online, it seems like a Windows .NET installation/version issue. Here's a [stackoverflow post](https://stackoverflow.com/questions/11432940/what-does-error-code-0xc0000135-or-1073741515-exit-code-mean-when-starting-a#:~:text=This%20error%20is%20caused%20when%20the%20.NET%20framework%20is%20not%20installed%20on%20the%20target%20computer%2C%20or%20when%20the%20version(s)%20installed%20are%20not%20sufficient%20to%20run%20the%20application.%20The%20resolution%20is%20obvious%3A%20install%20the%20correct%20version%20of%20the%20.NET%20framework%20before%20running%20the%20application.) with more info.
